strawberry/utils/typing.py

```diff
@@ -221,6 +222,11 @@ def _ast_replace_union_operation(
     return expr
 
 
+_annotated_re = re.compile(
+    r"(typing\.Annotated|Annotated\[)(?P<type>\w*),(?P<args>.*)(\])",
```

Comment:
> Maybe it would make more sense to just match Annotated and use re.search? People could alias typing import as t or as something else

that's a valid point!